src/index.html

```diff
@@ -16,17 +16,48 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <form
+      class="form"
+      action="#"
+      data-qa="big"
+      method="get"
+    >
+      <input
+        class="search search--big"
+        name="search-big"
+        data-qa="keypress"
+        type="text"
+        placeholder="Try “Los Angeles”"
+        id="big-field"
+      />
+      <label
+        class="form-field"
+        for="big-field"
+      >
+        label
+      </label>
```

Comment:
HTML Content: The `label` element should contain a description of the associated input field. This improves accessibility by providing a text description that screen readers can read out loud.